app/league/dashboard/views.py

```diff
@@ -173,7 +173,7 @@ def create_game():
             episode=form.episode.data,
             played_at=played_at
         )
-        messenger.notify_slack({'text':repr(game)})        
+        messenger.notify_slack(repr(game))
```

Comment:
The message is specific to game creation, and so should be part of `create_game`